src/ripple/app/rdb/impl/RelationalDBInterface_shards.cpp

```diff
@@ -32,6 +32,121 @@
 
 namespace ripple {
 
+DatabasePair
+makeMetaDBs(
+    Config const& config,
+    DatabaseCon::Setup const& setup,
+    DatabaseCon::CheckpointerSetup const& checkpointerSetup)
+{
+    // ledger meta database
+    auto lgr{std::make_unique<DatabaseCon>(
+        setup,
+        LgrMetaDBName,
+        LgrMetaDBPragma,
+        LgrMetaDBInit,
+        checkpointerSetup)};
+
+    if (config.useTxTables())
+    {
+        // transaction meta database
+        auto tx{std::make_unique<DatabaseCon>(
+            setup,
+            TxMetaDBName,
+            TxMetaDBPragma,
+            TxMetaDBInit,
+            checkpointerSetup)};
+
+        return {std::move(lgr), std::move(tx)};
+    }
+
+    return {std::move(lgr), {}};
+}
+
+bool
+saveLedgerMeta(
+    std::shared_ptr<Ledger const> const& ledger,
+    Application& app,
+    soci::session& lgrSesh,
+    soci::session& txnSesh,
+    std::uint32_t const shardIndex)
+{
+    std::string const lgrSQL =
```

Comment:
A constexpr string view may be a better option. There are a few other places that can also be replaced.
eg.
```    
    constexpr std::string_view lgrSQL =
        R"sql(INSERT OR REPLACE INTO LedgerMeta VALUES
              (:ledgerHash,:shardIndex);)sql";
```

Reply:
Done